terraform/digitalocean/project/variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "description" {
+  type        = string
+  description = "A description for this project"
+  default = ""
+}
+
+variable "resources" {
+  type        = list
+  description = "(Optional) List with all resources to be part of this project"
+  default     = []
+}
+
+variable "environment" {
+  type        = map
+  description = "(optional) Edit values below for every environment name that you want"
+  default = {
+    "dev" = "Development"
+    "stg" = "Staging"
+    "prd" = "Production"
+  }
+}
+
+variable "name" {
+  type        = string
+  description = "(Required) Name for your project at Digital Ocean"
+}
+
+variable "purpose" {
+  type        = string
+  description = "The purpose for your project. (for example: k8s or any), default - Web Application"
+  default = "Web Application"
+}
```

Comment:
please run `terraform fmt` 
And highly recommended [install pre-commit](https://github.com/shalb/cluster.dev/blob/master/docs/style-guide.md#style-guide) how it specified in our style-guide